config/application.rb

```diff
@@ -33,8 +38,5 @@ class Application < Rails::Application
     # We should consider changing how we do this.
     # Route exceptions to our custom error pages.
     config.exceptions_app = self.routes
-
-    # Do not swallow errors in after_commit/after_rollback callbacks.
-    config.active_record.raise_in_transactional_callbacks = true
```

Comment:
no longer valid https://stackoverflow.com/questions/28006358/undefined-method-raise-in-transactional-callbacks-for-activerecordbaseclass